dvc/stage/exceptions.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, missing_files):
         super().__init__(msg)
 
 
-class StageNotFound(KeyError, DvcException):
+class StageNotFound(DvcException):
```

Comment:
Why do we need to remove `KeyError`?

Reply:
```KeyError``` was the reason why we have extra quotes in the initial messages. To fix that, I've removed it. But before I double-checked the usage of ```StageNotFound```, and didn't find a reason for inheritance from ```KeyError```. So it looked safe for me to remove that inheritance.

Reply:
Internally, StageLoader still works like a dictionary. So, I'd still prefer that we keep the semantics of `StageNotFound` being a KeyError as well.

Here's one example:
https://github.com/iterative/dvc/blob/273518afab02a7b229fc0ef0eba43a556fb80a44/dvc/repo/stage.py#L184

We can get around the quoting behaviour by using a custom `__str__` implementation.
```suggestion
    __str__ = DvcException.__str__
```

Reply:
Done